### PR TITLE
Largest series product

### PR DESCRIPTION
**Pull request by vrnithinkumar**
Adding test case generator for largest series product . Fix for #382.
- Added "digit" as a properties with Identifier since it is very long some times.
- Added property "comment" in excluded list.
- Fixed example to handle invalid cases.

### CODE REVIEW

File: exercises/largest-series-product/LargestSeriesProductTest.fs [hunk 1]
```diff
@@ -1,40 +1,84 @@
...
+
+[<Fact(Skip = "Remove to run test")>]
+let ``Reports 1 for nonempty string and empty product (0 span)`` () =
+    let digits = "123"
+    largestProduct digits 0 |> should equal 1
+
+[<Fact(Skip = "Remove to run test")>]
+let ``Rejects empty string and nonzero span`` () =
+    let digits = ""
+    largestProduct digits 1 |> should equal -1
```
Comment (ErikSchierboom) [inline exercises/largest-series-product/LargestSeriesProductTest.fs]: I would like to encode an error as `None`.

File: generators/Generators.fs [hunk 2]
```diff
@@ -154,6 +154,11 @@ type KindergartenGarden() =
 
     override this.UseFullMethodName canonicalDataCase = true
 
+type LargestSeriesProduct() =
+    inherit Exercise()
+
+     override this.PropertiesWithIdentifier canonicalDataCase = ["digits"]
```
Comment (ErikSchierboom) [inline generators/Generators.fs]: Good idea to have this as an identifier!

File: exercises/largest-series-product/LargestSeriesProductTest.fs (hunk 1 above)
Comment (vrnithinkumar) [reply exercises/largest-series-product/LargestSeriesProductTest.fs]: Made changes to test use `int option` as return type and use `None` for error case.

**Comment by ErikSchierboom** [comment]
Merged. Thanks!

**Pull request merged**